FirebaseAppDistribution/Sources/FIRAppDistribution.m

```diff
@@ -339,7 +339,7 @@ - (BOOL)isCodeHashIdentical:(NSString *)codeHash {
   return codeHash && [codeHash isEqualToString:[machO codeHash]];
 }
 
-- (void)startFeedback:(NSString *)infoText {
+- (void)startFeedbackWithInfoText:(NSString *)infoText {
```

Comment:
Shouldn't this and the methods below be in the form `*WithAdditionalFormText` ?

Reply:
Thanks for catching this!